tests/migration/test_20231124_1c14468b74ce.py

```diff
@@ -26,7 +26,7 @@ def test_migration(
     create_identifier: CreateIdentifier,
     create_license_pool: CreateLicensePool,
 ) -> None:
-    alembic_runner.migrate_up_to(MIGRATION_UID)
+    alembic_runner.migrate_down_to(MIGRATION_UID)
```

Comment:
This was an error in the existing test that only manifested when another migration was added.